flare_flutter/lib/flare_actor.dart

```diff
@@ -20,8 +21,9 @@ typedef void FlareCompletedCallback(String name);
 /// Several constructors are provided for the various ways that a Flare can be
 /// specified:
 ///  * [FlareActor], for obtaining a Flare from an asset [filename].
-///  * [FlareActor.asset], for obtaining a Flare from an [AssetBundle]
-///    using a key.
+///  * [FlareActor.asset], for obtaining a Flare from an [AssetProvider].
+///  * [FlareActor.rootBundle], for obtaining a Flare from an [AssetBundle].
```

Comment:
Tiniest of nitpicks: I thought the purpose of this constructor was specifically for obtaining the file from the root asset bundle. Should we change the comment to reflect or?

Otherwise, if the intention is to allow a generic bundle, maybe we should change the signature to allow for that (and default to the root bundle if not provided). Maybe also rename it back to FlareActor.bundle.  What do you think?